e2e/utils/departures.ts

```diff
@@ -1,7 +1,7 @@
 import {by, element, expect} from 'detox';
 import {tapById} from './interactionHelpers';
 import {expectIdToHaveText, expectToBeVisibleByText} from './expectHelpers';
-import {chooseSearchResult, setInputById} from './commonHelpers';
+import { chooseSearchResult, idExists, setInputById } from "./commonHelpers";
```

Comment:
Is your linter configured correctly? This line seems inconsistent with the rest of the file.

Reply:
Looks like I forgot to run it on the changed file... My bad!